source/connection-troubleshooting.txt

```diff
@@ -204,6 +204,43 @@ Consider setting ``maxConnectionLifeTime`` and
 maintains a connection to a MongoDB instance. For more information about these
 parameters, see :ref:`Connection Pool Settings <mcs-connectionpool-settings>`.
 
+.. _java-server-selection-errors:
+
+Server Selection Errors
+-----------------------
+
+MongoDB might not be able to complete a request even when some servers
+are available, causing the driver to return a server selection error.
+The following list describes some situations in which you
+might encounter a server selection error:
+
+- You use tag sets in a read preference specification when no server
```

Comment:
This seems more like an application bug.  It's more likely that no servers with those tags are reachable from the application at this time, either due to a network error or database server maintenance.

Reply:
Removed this list